modules/qualitycontrol/ReceiveVideoController.js

```diff
@@ -133,8 +170,16 @@ export default class ReceiveVideoController {
          */
         this._sourceReceiverConstraints = new Map();
 
+        /**
+         * The number of bps requested from the bridge.
+         */
+        this._assumedBandwidthBps = sanitizeAssumedBandwidthBps(config?.assumedBandwidthBps);
```

Comment:
Let's default to -1 and not use config, I don't think it makes much sense to put a config like that, wholesale...